java/registry/src/main/java/io/opensaber/registry/shard/advisory/SerialNumberShardAdvisory.java

```diff
@@ -0,0 +1,30 @@
+package io.opensaber.registry.shard.advisory;
+
+import org.springframework.stereotype.Component;
+
+import io.opensaber.registry.model.DBConnectionInfo;
+import io.opensaber.registry.model.DBConnectionInfoMgr;
+
+@Component
+public class SerialNumberShardAdvisory implements IShardAdvisory {
+
+	private DBConnectionInfoMgr dBConnectionInfoMgr;
+	
+	public SerialNumberShardAdvisory(DBConnectionInfoMgr dBConnectionInfoMgr){
+		this.dBConnectionInfoMgr = dBConnectionInfoMgr;
+	}
+	
+	@Override
+	public DBConnectionInfo connectionInfo(String subject) {
```

Comment:
please rename subject as serialNumber to make it clear.

Reply:
Done